modules/nodes-nodes-working-setting-booleans.adoc

```diff
@@ -0,0 +1,56 @@
+[id="nodes-nodes-working-setting-booleans"]
+
+= Setting SELinux booleans
+
+{product-title} allows you to enable and disable a SELinux boolean on a RHEL CoreOC node. The following procedure explains how to modify SELinux booleans on nodes using the MachineConfigOperator (MCO). The `container_manage_cgroup` is the example sebool in this procedure. You can modify this value to whichever sebool you need.
```

Comment:
s/ RHEL / {op-system-first}

Reply:
Actually, I think `{product-title}` is correct here. But "RHEL CoreOC" should be "Red Hat Enterprise Linux CoreOS (RHCOS)" (which should be replaced in the .adoc as `op-system-first`). In other words, the intro sentence, IIUC, should render as:
"OpenShift Container Platform allows you to enable and disable an SELinux boolean on a Red Hat Enterprise Linux CoreOS (RHCOS) node." Correct?